docs/sources/alerting/set-up/_index.md

```diff
@@ -0,0 +1,17 @@
+---
+aliases:
+  - /docs/grafana/latest/alerting/set-up/
+  - /docs/grafana/latest/alerting/set-up/
+  - /docs/grafana/latest/alerting/unified-alerting/set-up/
+title: Set up Grafana Alerting
+weight: 107
+---
+
+# Set up Grafana Alerting
+
+Configure the features and integrations that you need to create and manage your alerts.
+
+- [Configure Alertmanager](https://grafana.com/docs/grafana/latest/alerting/set-up/configure-alertmanager/)
+- [Provision Grafana Alerting resources](https://grafana.com/docs/grafana/latest/alerting/set-up/provision-alerting-resources/)
+
```

Comment:
Why is there a blank line here?

Reply:
Very nitty. I kinda wanted to separate out the config steps and the integrations, but happy to put together until this gets beefed out and maybe gets two separate mini headings or so

Reply:
Updated:)